src/accelerate/accelerator.py

```diff
@@ -365,7 +365,10 @@ def __init__(
             and self.distributed_type != DistributedType.FSDP
             and self.distributed_type != DistributedType.MEGATRON_LM
         ):
-            self.native_amp = is_bf16_available(True)
+            if self.device.type == "cpu":
+                self.native_amp = is_torch_version(">=", "1.10")
+            else:
+                self.native_amp = is_bf16_available(True)
```

Comment:
I don't understand the need for this, `is_bf16_available` already does this check as shown here: https://github.com/huggingface/accelerate/blob/6ebaab71bbf87960583bc17339c9da9ec2e4ea91/src/accelerate/utils/imports.py#L89-L92